ftw/simplelayout/aliasblock/contents/aliasblock.py

```diff
@@ -0,0 +1,24 @@
+from ftw.simplelayout import _
+from ftw.simplelayout.aliasblock.contents.interfaces import IAliasBlock
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.dexterity.content import Item
+from plone.directives import form
+from zope import schema
+from zope.interface import alsoProvides
+from zope.interface import implements
+
+
+class IAliasBlockSchema(form.Schema):
+    """AliasBlock for simplelayout
+    """
+
+    title = schema.TextLine(
```

Comment:
@busykoala Please also add the show_title bool field, so we have more consistent behavior for the different blocks.

Reply:
The field is removed in further implementation. There will be no title field as specified...